### DISCUSSION TITLE
Field for a point on a map

### DISCUSSION THREAD

**Discussion by emme1444** [keystonejs/keystone#5756] [Feature requests]
**Field for a point on a map.**

The field would contain something like:
- _label_: `string` - for labeling the point
- _longitude_: `decimal` - for storing the longitude of the point on the map
- _latitude_: `decimal` - for storing the latitude of the point on the map

Preferably with the ability to choose from a map within the UI, although this might become too much? Or if there is any other way to somehow integrate the ability to easily choose the point from a map. Maybe just grabbing the label of the point using some API (Google).

Would this be something doable?

**Comment by MurzNN** [comment]
Keystone uses Prisma as database driver, which is missing support for that type of data, feature request is here: https://github.com/prisma/prisma/issues/5984
As alternative, you can use JSON field with those fields as one object, or separate `decimal` fields in List for each singular value.

**Comment by MurzNN** [comment]
Also in Keystone 5 there was the separate field type with widget in Admin UI https://github.com/keystonejs/keystone-5/tree/main/packages/fields-location-google - maybe developers will port it to Keystone 6 in near future...

**Comment by raveling** [comment]
@MurzNN 's comment about Prisma support remains accurate.
If/when that changes we will  take another look.
Thanks all!
